Formula/p/phylum-cli.rb

```diff
@@ -21,12 +21,33 @@ class PhylumCli < Formula
 
   uses_from_macos "zlib"
 
+  # The `--no-default-features` option is used for Cargo commands to ensure the
+  # Phylum CLI default feature "selfmanage" is not activated. That feature allows
+  # for updating and uninstalling the CLI, which is undesired in this context:
+  # https://docs.brew.sh/Acceptable-Formulae#we-dont-like-tools-that-upgrade-themselves
   def install
-    system "cargo", "install", *std_cargo_args(path: "cli")
+    system "cargo", "install", "--no-default-features", *std_cargo_args(path: "cli")
+
+    # Generate and install shell completions
+    system "cargo", "run", "--package", "xtask", "--no-default-features", "gencomp"
+    bash_completion.install "target/completions/phylum.bash"
+    zsh_completion.install "target/completions/_phylum"
+    fish_completion.install "target/completions/phylum.fish"
+  end
+
+  # Default official extensions are not installed because the standard install
+  # script asks the user for permission to overwrite existing extensions and
+  # blocking on user input is not desired in the context of a package manager.
+  def caveats
+    <<~EOS
+      No official extensions have been installed.
+      For a list of official extensions with installation instructions, see:
+        https://github.com/phylum-dev/cli/tree/main/extensions
+    EOS
   end
```

Comment:
I feel like this can be removed as well as it is not homebrew specific

Reply:
For the homebrew specific part we can reduce it to:
```suggestion
  def caveats
    <<~EOS
      No official extensions have been preinstalled.
    EOS
  end
```